md/product-versioning.md

```diff
@@ -3,12 +3,24 @@ Discover how Bonita manages its product and artifacts versions and how it can im
 
 ## Product Versions
 
-Bonita follows the usual semantic versioning X.Y.z:
+After years of semantic versioning, Bonita now follows the calendar versioning: YYYY.#-mmdd, since version 2021.1:
+* YYYY is the year of the release
+* \# is the release number. Two versions are released every year.
+* mmdd is the month and day of the release. A new maintenance version of the released number is made available every month. For example the february version will be 2021.1-0205.Keep in mind that all maintenance version are cross-compatible (see the Technical Id chapter)
```

Comment:
```suggestion
* mmdd is the month and day of the release. A new maintenance version is made available every month. For example, the February version will be 2021.1-0205 as it will be released on the 5th. Keep in mind that all maintenance versions are cross-compatible (see the Technical Id chapter).
```